aiohttp/connector.py

```diff
@@ -287,7 +291,7 @@ def __enter__(self) -> 'BaseConnector':
         return self
 
     def __exit__(self, *exc: Any) -> None:
-        self.close()
+        self._close()
```

Comment:
so now `_close` is synchronous, while `close` is asynchronous

Reply:
Ok. We should drop sync `__enter__` / `__exit__` in the following PR, they are not needed anymore.
Even worse: `with connector:` statement becomes dangerous.

Reply:
Well, drop it now please

Reply:
ok sure